### PR TITLE
Add userfees, userfundings, and spotclearinghousestate to info client

### PR DESCRIPTION
**Pull request by sree-revoori1**
Also, fix github workflow to use the master branch instead of main.

### CODE REVIEW

File: src/bin/info.rs [hunk 1]
```diff
@@ -55,6 +62,30 @@ async fn user_states_example(info_client: &InfoClient) {
     );
 }
 
+async fn user_token_balances_example(info_client: &InfoClient) {
+    let user: String = "0xc64cc00b46101bd40aa1c3121195e85c0b0918d8"
```
Comment (lmlmt) [inline src/bin/info.rs:66]: nit: define address string as a constant and reuse

File: src/info/info_client.rs [hunk 2]
```diff
@@ -139,6 +161,22 @@ impl InfoClient {
         serde_json::from_str(&return_data).map_err(|e| Error::JsonParse(e.to_string()))
     }
 
+    pub async fn user_token_balances(&self, address: H160) -> Result<UserTokenBalanceResponse> {
+        let input = InfoRequest::UserTokenBalances { user: address };
```
Comment (lmlmt) [inline src/info/info_client.rs:165]: a lot of this code is copy-paste, can you define an inner, non-pub function that takes in `InfoRequest` along with `address` and returns a `Result<T>`?

Comment (sree-revoori1) [reply src/info/info_client.rs:167]: Done

File: src/bin/info.rs (hunk 1 above)
Comment (sree-revoori1) [reply src/bin/info.rs]: Done

File: src/bin/info.rs [hunk 3]
```diff
@@ -17,13 +18,17 @@ async fn main() {
...
+    query_order_by_oid_example(&info_client).await;
+    query_referral_state_example(&info_client).await;
 }
 
 async fn open_orders_example(info_client: &InfoClient) {
-    let user: String = "0xc64cc00b46101bd40aa1c3121195e85c0b0918d8"
-        .parse()
-        .unwrap();
-    let user = H160::from_str(&user).unwrap();
+    let user = ADDRESS.to_owned().parse::<H160>().unwrap();
```
Comment (lmlmt) [inline src/bin/info.rs:31]: nit: just define a function address and use that everywhere to further reduce copy-paste, i.e.
```
fn address() -> H160 {
  "0xc64cc00b46101bd40aa1c3121195e85c0b0918d8".to_string().parse().unwrap()
}
```

**Review by lmlmt — APPROVED** [review]
looks good, other than the one comment

File: src/bin/info.rs (hunk 3 above)
Comment (sree-revoori1) [reply src/bin/info.rs]: Done

**Review by lmlmt — APPROVED** [review]

**Pull request merged**